napari/_qt/containers/_layer_delegate.py

```diff
@@ -93,7 +93,10 @@ def get_layer_icon(self, option, index):
         else:
             icon_name = f'new_{layer._type_string}'
 
-        icon = QColoredSVGIcon.from_resources(icon_name)
+        try:
+            icon = QColoredSVGIcon.from_resources(icon_name)
+        except ValueError:
+            return
```

Comment:
Do we want no icon here, or maybe a default one? We can always add a default later, so no need to hold up PR, but curious what others think

Reply:
I'm going to merge this PR now, we can always add do this later